actions/gerald-pr/action.yml

```diff
@@ -25,7 +25,7 @@ runs:
           ref: '${{ github.head_ref }}'
         if: ${{ github.event.pull_request.draft == false }}
       - name: Run Gerald
-        uses: Khan/gerald@v4.0
+        uses: Khan/gerald@v4.1
```

Comment:
It may be better to just change this to `v4` so that minor updates will get rolled out automatically.

Reply:
ooooh... I didn't know it can do that! That's interesting... I would have thought that anything after the `@` was just an opaque git ref. What part of the system understands semantic versioning embedded in git tags?

Reply:
Ah - you know what, I take it back! So this only works for things published in this repo. Gerald isn't set up to do this yet. We have to hardcode the version here... ☹️ If actions here are published you can reference them by major version, which is nice!